qcengine/programs/torchani.py

```diff
@@ -103,15 +105,19 @@ def compute(self, input_data: "AtomicInput", config: "TaskConfig") -> "AtomicRes
         ret_data = {"success": False}
 
         # Build model
-        model = self.get_model(input_data.model.method)
-        if model is False:
-            raise InputError("TorchANI only accepts the ANI1x or ANI1ccx method.")
+        method = input_data.model.method
+        model = self.get_model(method)
 
         # Build species
-        species = "".join(input_data.molecule.symbols)
-        unknown_sym = set(species) - {"H", "C", "N", "O"}
+        species = input_data.molecule.symbols
```

Comment:
`molecule.symbols` will return e.g., `Cl` (capitalize case). does the below need adjusting?

Reply:
Sorry for the delayed response! I pushed your commit for `Cl`. Thanks